src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5636,30 +5644,18 @@ namespace EnergyPlus::DaylightingManager {
         // na
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 GTOT;  // Glare constant
-        Real64 GTOT1; // Portion of glare constant
-        Real64 GTOT2; // Portion of glare constant
-        int IWin;     // Window counter
-        int IS;       // Window shading index: 1=unshaded, 2=shaded
-        int loop;     // Loop index
-
-        // FLOW:
-        // Initialize glare constant
-        GTOT = 0.0;
+        Real64 GTOT  = 0.0;  // Glare constant
 
         // Loop over exterior windows associated with zone
-        for (loop = 1; loop <= state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins; ++loop) {
-            IWin = state.dataDaylightingData->ZoneDaylight(ZoneNum).DayltgExtWinSurfNums(loop);
-            IS = 1;
-            if ((SurfWinWindowModelType(IWin) != WindowBSDFModel) &&
-                ((SurfWinShadingFlag(IWin) >= 1 && SurfWinShadingFlag(IWin) <= 9) || SurfWinSolarDiffusing(IWin)))
-                IS = 2;
+        for (int loop = 1; loop <= state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins; ++loop) {
+            int IWin = state.dataDaylightingData->ZoneDaylight(ZoneNum).DayltgExtWinSurfNums(loop);
+            int WinShadingIndex = findWinShadingIndex(IWin);
```

Comment:
Hmm, 'WinShadingIndex' implies more than just two states, 1=unshaded, 2=shaded. I guess the name of this doesn't matter as long at `findWinShadingIndex` has a comment to explain this is simply 1=unshaded, 2=shaded.